tracer.go

```diff
@@ -304,6 +306,7 @@ func (t *Tracer) TracePrepareStart(ctx context.Context, conn *pgx.Conn, data pgx
 	opts := []trace.SpanStartOption{
 		trace.WithSpanKind(trace.SpanKindClient),
 		trace.WithAttributes(t.attrs...),
+		trace.WithAttributes(PrepareStmtNameKey.String(data.Name)),
 	}
```

Comment:
minor: AFAIK we still can have non-prepared statements, so maybe we could only add this if the statement has a name?

```suggestion
	}
	
	if data.Name != "" {
		trace.WithAttributes(PrepareStmtNameKey.String(data.Name)),
	}
```